lms/djangoapps/completion_api/views.py

```diff
@@ -103,12 +228,122 @@ def get(self, request):
 
 
 class CompletionDetailView(APIView, CompletionViewMixin):
+    # pylint: disable=line-too-long
     """
-    API view to render serialized CourseCompletions.
+    API view to render a serialized CourseCompletion for a single user in a
+    single course.
+
+    **Request Format**
+
+        GET /api/completion/v0/course/<course_key>/
+
+    **Example Requests**
+
+        GET /api/completion/v0/course/course-v1:GeorgetownX+HUMX421-02x+1T2016/
+        GET /api/completion/v0/course/course-v1:edX+DemoCourse+Demo2017/?requested_fields=chapter,vertical
+
+    **Response Values**
+
+        Standard fields:
+
+        * course_key (CourseKey): The unique course identifier.
+        * completion: A dictionary comprising of the following fields:
+            * earned (float): The sum of the learner's completions.
+            * possible (float): The total number of completions available
+              in the course.
+            * ratio (float in the range [0.0, 1.0]): The ratio of earned
+              completions to possible completions in the course.
+
+        Optional fields:
+
+        * If "requested_fields" is specified, the response will include data
+          for specific block types.  The fields available are configurable, but
+          may include `chapter`, `sequential`, or `vertical`.  If requested,
+          the block type will be present as another field in the response.
+          Inside the field will be a list of all blocks of that type containing
+          completion information for that block.  Fields for each entry will
+          include:
+
+              * course_key (CourseKey): The unique course identifier.
+              * usage_key: (UsageKey) The unique block identifier.
+              * completion: A dictionary comprising the following fields.
```

Comment:
Optional: You could just say "see above" here, since the format is the same as `completion` described under "Standard Fields".